app/styles/_membership.less

```diff
@@ -79,10 +80,12 @@
       }
     }
   }
+  .half-width {
+    min-width: 50% !important;
```

Comment:
Its a class conditionally set:
```html
ng-class="{ 'half-width': !mode.edit }"
```
Setting `flex-basis: 50%` doesn't have the same effect.  There may be a better way to do it, but it is descriptive of my intent.  

Thinking on it...